patches/minecraft/net/minecraft/client/gui/screens/advancements/AdvancementsScreen.java.patch

```diff
@@ -21,7 +21,7 @@
 +          m_142416_(net.minecraft.client.gui.components.Button.m_253074_(Component.m_237113_(">"), b -> tabPage = Math.min(tabPage + 1, maxPages))
 +              .m_252794_(guiLeft + f_169556_ - 20, guiTop - 50).m_253046_(20, 20).m_253136_());
 +          maxPages = this.f_97335_.size() / AdvancementTabType.MAX_TABS;
-+      }
++     }
```

Comment:
If you would allow me to pester you one more time :)

Reply:
That was a space that was added in previous commit and was removed in the next commit to fix the prior issues

https://github.com/MinecraftForge/MinecraftForge/blob/1.20.x/patches/minecraft/net/minecraft/client/gui/screens/advancements/AdvancementsScreen.java.patch

Looks the same compared to this in terms of spacing for line 24 when comparing the 2nd commit to it.

Reply:
Works for me